akka-http/src/main/scala/akka/http/javadsl/server/directives/WebSocketDirectives.scala

```diff
@@ -44,17 +44,16 @@ abstract class WebSocketDirectives extends SecurityDirectives {
    * Handles WebSocket requests with the given handler and rejects other requests with an
    * [[ExpectedWebSocketRequestRejection]].
    */
-  def handleWebSocketMessages[T](handler: Flow[Message, Message, T]): Route = RouteAdapter {
+  def handleWebSocketMessages(handler: Flow[Message, Message, NotUsed]): Route = RouteAdapter {
```

Comment:
Yeah, but now it forces the flow to be a materialized value of type `NotUsed` while before it could be whatever and be ignored. The Scala signature does not need the parameter because of declaration site variance, which Java does not have. I'm pretty sure this is why there was a difference to start with. 

I think it should stay like how it was, you know, _Be conservative in what you do, be liberal in what you accept from others_, having to do a `mapMaterializedValue` just to get the right ignored type is not nice!